app/main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
it's global class attribute, thus whenever you add some skills here, then all class instances will have it, but this is incorrect